projects/mariadb/fuzz_get.c

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 Google LLC
+// Modifications copyright (C) 2023 ISP RAS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <tap.h>
+#include <my_sys.h>
+#include <json_lib.h>
+
+#define FUZZ_KEY_SIZE 32
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  if (size < FUZZ_KEY_SIZE) {
+    return -1;
+  }
+  char *fuzz_key = malloc(FUZZ_KEY_SIZE + 1);
+  memcpy(fuzz_key, data, FUZZ_KEY_SIZE);
+  fuzz_key[FUZZ_KEY_SIZE] = '\0';
+
+  data += FUZZ_KEY_SIZE;
```

Comment:
add valid corpus files according to data format (key + json)

Reply:
added